microprofile-jwt/pom.xml

```diff
@@ -15,6 +15,8 @@
         <dependency>
             <groupId>org.eclipse.microprofile</groupId>
             <artifactId>microprofile</artifactId>
+            <!-- TODO: remove once all microprofile specs will be updated to MP4 -->
+            <version>4.0.1</version>
```

Comment:
Just a note - if you rebase against current MP4 branch, you should be able to get rid of this already since there is performed update for all modules.